README.md

```diff
@@ -19,9 +19,9 @@ Encourage and help software developers set up their releases to be fully automat
 # shipkit-auto-version Gradle Plugin
```

Comment:
Can you also update the "shipkit-auto-version Gradle Plugin" section? That section indicates that you *need* a `version.properties` file. Perhaps we can simplify this section to avoid duplication with "Usage" section.